adam-cli/src/main/scala/org/bdgenomics/adam/cli/TransformGenotypes.scala

```diff
@@ -135,7 +137,15 @@ class TransformGenotypes(val args: TransformGenotypesArgs)
     if (args.outputPath.endsWith(".vcf")) {
       maybeSort(maybeCoalesce(genotypes.toVariantContexts)).saveAsVcf(args)
     } else {
-      maybeSort(maybeCoalesce(genotypes)).saveAsParquet(args)
+      if (args.partitionedBinSize > 0) {
+        if (genotypes.sequences.isEmpty) {
+          log.warn("This dataset is not aligned and therefore will not benefit from being saved as a partitioned dataset")
```

Comment:
Nit: Genotypes are aligned by definition, no? I don't have a strong preference as to what we do here, but this warning just reads a bit odd.